### PR TITLE
Update for 0.57

### PR DESCRIPTION
**Pull request by lukastaegert**
Document new flags and configuration options

### CODE REVIEW

File: guide/en/03-command-line-reference.md [hunk 1]
```diff
@@ -129,6 +132,24 @@ $ rollup --config
...
+
+export default commandLineArgs => {
+  if (commandLineArgs.debug === true) {
+    return debugConfig;
+  }
+  return defaultConfig;
+}
+```
+
+If you now run `rollup --config --debug`, the debug configuration will be used. Note that it is your responsibility to avoid conflicts with existing rollup options.
```
Comment (guybedford) [inline guide/en/03-command-line-reference.md]: As mentioned in https://github.com/rollup/rollup/pull/1926 I'm not sure we should publicise this feature yet without discussing the namespacing implications a little further.

File: guide/en/04-javascript-api.md [hunk 2]
```diff
@@ -87,6 +89,7 @@ const outputOptions = {
   sourcemapFile,
   interop,
   extend,
+  namespaceToStringTag,
```
Comment (guybedford) [inline guide/en/04-javascript-api.md]: Maybe worth putting this in danger zone for now (it's directly related to `freeze` as well in acting on namespaces).

**Comment by lukastaegert** [comment]
**Update**: This now reflects the latest changes in rollup/rollup#2063 + review comments. I also added a clear distinction for output options in the big option list. @guybedford Please have another look.

File: guide/en/03-command-line-reference.md [hunk 3]
```diff
@@ -22,18 +22,19 @@ export default { // can be an array (for multiple inputs)
...
   treeshake,
   context,
   moduleContext,
-  legacy,
   
   // experimental
-  experimentalDynamicImport,
   experimentalCodeSplitting,
+  experimentalDynamicImport,
+  experimentalPreserveModules,
```
Comment (guybedford) [inline guide/en/03-command-line-reference.md]: Can we keep preserveModules undocumented for a little longer? I think I may have spotted some more inconsistencies with the pathing today.

File: guide/en/03-command-line-reference.md [hunk 4]
```diff
@@ -59,7 +60,9 @@ export default { // can be an array (for multiple inputs)
     amd,
     indent,
     strict,
-    freeze
+    freeze,
+    legacy,
+    namespaceToStringTag
```
Comment (guybedford) [inline guide/en/03-command-line-reference.md]: 👍

File: guide/en/03-command-line-reference.md (hunk 3 above)
Comment (lukastaegert) [reply guide/en/03-command-line-reference.md]: Done! To put it back in, we can just revert the last commit once you think it's ready.

**Pull request merged**